WikiWatershed/model-my-watershed
# MapShed Compare: Window and Scenario Header Row

## Overview

Enables Compare View for GWLF-E projects. Adds colors for 32 scenarios to be used in the charts. Adds a header row for GWLF-E scenarios that don't have a map, but do have a color badge. The popover works the same as in TR-55, but the modifications aren't listed correctly. That is left for #2920.

Connects #2908 
Connects #2912 
Connects #2913 

### Demo

![image](https://user-images.githubusercontent.com/1430060/44053760-9ce2eac6-9f0e-11e8-9fb0-d1c7ff13eabf.png)

![2018-08-13 15 37 18](https://user-images.githubusercontent.com/1430060/44053853-e27a895e-9f0e-11e8-9918-85882ce7fb1d.gif)

## Testing Instructions

* Check out this branch and `bundle`
* Make a GWLF-E project. Add a scenario. Try the Compare View. Ensure you see a colored badge against each Scenario's name in the header, and that there is no map.
* Make a TR-55 project. Add a scenario. Try the Compare View. Ensure works same as before.
